### PR TITLE
Add first version of download page for idr.openmicroscopy.org

### PR DESCRIPTION
**Pull request by sbesson**
Initial version is composed of
- a short description of the Aspera server
- both manual and Docker instructions
- a paragraph explaining how to find relative paths

### CODE REVIEW

**Comment by sbesson** [comment]
Staged at https://snoopycrimecop.github.io/idr.openmicroscopy.org/download.html

File: _includes/aspera.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+- server: fasp.ebi.ac.uk
+- port: 33001
+
+Each published study has an associated passwordless username matching the IDR
+accession number e.g. `idr0001`.
+
+## Manual instructions
+
+The Aspera command line client can be downloaded  [here](http://www.asperasoft.com/downloads/connect). The following command
```
Comment (mtbc) [inline _includes/aspera.md]: double-space!  though maybe wrap this long line while here

File: _includes/aspera.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+will download all the raw data associated with the `idr0008` submission:
+
+    $ ascp -TQ -l40m -P 33001 -i "path/to/asperaweb_id_dsa.openssh" idr0008@fasp.ebi.ac.uk:. /tmp/data/idr0008/
+
+Refer to the [Aspera documentation](https://downloads.asperasoft.com/documentation/)
+for more details about the command line options.
```
Comment (mtbc) [inline _includes/aspera.md]: "command-line options"

File: _includes/aspera.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+## Docker instructions
+
+For convenience, a
+[Docker image](https://hub.docker.com/r/imagedata/download)
+iis available with the IDR download options. You can use it as follows:
```
Comment (mtbc) [inline _includes/aspera.md]: is

File: download.html [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+        <header class="marketing-hero">
+            <div class="row homepage text-center">
+
+                <div class="medium-12 columns">
+
+                    <h1 class="hero-main-header">Data download</h1>
+                    <p class="hero-subheader small-10 medium-10 large-10 small-offset-1 medium-offset-1 large-offset-1">
+                    The raw imaging data for all studies published in the
+                    Image Data Resource (IDR) is available for download through
+                    the Aspera protocols.
```
Comment (mtbc) [inline download.html]: Should "protocol" be singular?

**Comment by dominikl** [comment]
I was just trying to follow the doc. What are these last parameters `.` and `/data`?

Running for example:
`docker run -ti --rm -v /tmp/data/idr0021:/data imagedata/download idr0021 . /data`

... just leads to a lot of error message:
```
...
siCTRL_PCNT_NEDD1_20100507_Fri-1643_0_SIR_PRJ                                                                                                100%  515KB            - error -
                                                                                                                         Error 50 [Destination: No such file or directory]
Partial Completion: 0K bytes transferred in 0 seconds
 (0K bits/sec), in 415 files, 40 directories; 415 files failed; 39 dir creations failed.

Session Stop  (Error: One or more transfers failed)
```

**Comment by dominikl** [comment]
Works when the directory permissions are set to 777, thanks @sbesson . Wonder if that should be mentioned in the doc as well.

**Comment by sbesson** [comment]
Definitely worth documenting. As an alternative to the very scary 777 permissions, the ID of the default user of the Docker image is `1000`. So, 

```
mkdir -p /tmp/data/idr0021
chown -R 1000:1000 /tmp/data/idr0021
docker run -ti --rm -v /tmp/data/idr0021:/data imagedata/download idr0021 . /data
```

should work in principle. Can you give that a try and maybe we propose this workflow ?

**Comment by joshmoore** [comment]
Does it help (has it been tested) to run the docker container with the current user's ID?

**Comment by sbesson** [comment]
Testing with my userid gives,

```
[sbesson@idr1-slot2 ~]$ docker run --rm -it -u 11615:11615 -v idr0001:/data download idr0001 md5sum.txt /data
/home/data/.aspera/connect/bin/ascp -P33001 -l 40m -i /root/.aspera/connect/etc/asperaweb_id_dsa.openssh idr0001@fasp.ebi.ac.uk:md5sum.txt /data
/usr/local/bin/ascp.sh: line 22: /root/.aspera/connect/bin/ascp: Permission denied
```

I suspect modifications might be required in the `imagedata/download` image but I will not have the time to drive this.

I will come back and try to complete this next week comparing with  the Aspera instructions of other existing EBI archives  ([EGA](https://www.ebi.ac.uk/ega/about/ftp-aspera#aspera_3), [ENA](https://www.ebi.ac.uk/ena/browse/read-download#downloading_files_aspera), [PRIDE](https://www.ebi.ac.uk/pride/help/archive/asperadownload), [EMPIAR](https://www.ebi.ac.uk/pdbe/emdb/empiar/faq)).

**Comment by sbesson** [comment]
Thanks @manics. Merging for inclusion in `prod65`. Improvements can follow-up to cover Docker permissions, graphical client workflow...

**Pull request merged**